mlflow/utils/rest_utils.py

```diff
@@ -183,6 +186,41 @@ def verify_rest_response(response, endpoint):
     return response
 
 
+def _validate_retries_and_backoff_values(max_retries, backoff_factor):
+    max_retry_limit = _MLFLOW_HTTP_REQUEST_MAX_RETRIES_LIMIT.get()
+    max_backoff_factor_limit = _MLFLOW_HTTP_REQUEST_MAX_BACKOFF_FACTOR_LIMIT.get()
+
+    if max_backoff_factor_limit < 0:
+        raise MlflowException(
+            "The current maximum backoff factor limit is invalid "
+            f"({max_backoff_factor_limit}). Cannot be negative."
+        )
+    if max_retry_limit < 0:
+        raise MlflowException(
+            f"The current maximum retry limit is invalid ({max_retry_limit}). "
+            "Cannot be negative."
+        )
+    if max_retries >= max_retry_limit:
+        raise MlflowException(
+            f"The configured max_retries ({max_retries}) are "
+            f"in excess of the maximum allowable retries ({max_retry_limit})"
+        )
+    if backoff_factor >= max_backoff_factor_limit:
+        raise MlflowException(
+            f"The configured backoff_factor ({backoff_factor}) are in excess "
```

Comment:
```suggestion
            f"The configured backoff_factor ({backoff_factor}) is in excess "
```

should we use `is` here?